### DISCUSSION TITLE
v1.4.3

### DISCUSSION THREAD

**Discussion by ggerganov** [ggml-org/whisper.cpp#1446] [Announcements]
This is a minor release, the main reason for which is that there hasn't been an official release for a few months now and some small things have accumulated on the `master` branch that would be nice to be upstreamed. I am planning a major `v1.5.0` release with some new and long-waited functionality soon:

- Full CUDA offloading
- Efficient Beam-Search implementation
- Grammar support

The current version `v1.4.3` should be considered in beta as I haven't worked intensively on `whisper.cpp` recently and there might be some issues that made their way in the code. I'll try to polish things in the next days and prepare a stable `v1.5.0` release. In the meantime, any feedback will be highly appreciated.

***Detailed API changes, features and new contributor recognitions will be included in the `v1.5.0` release.***

<hr /><em>This discussion was created from the release <a href='https://github.com/ggerganov/whisper.cpp/releases/tag/v1.4.3'>v1.4.3</a>.</em>

**Comment by npaine01** [comment]
Will 1.5 have whisper v3?

Also, will this be something that is updated on the paid versions of the apps as well, or is this just the GitHub downloadable versions?

**Reply by npaine01** [reply]
Also, I'm not sure how this works at all, but is there any way to have the software distinguish between speakers?

**Reply by ggerganov** [reply]
v3 support is already available in `whisper.cpp`. Support in 3rd party apps using `whisper.cpp` depends on their developers and we have no control over this. There is no support for distinguishing speakers.
